project/src/components/city-item-list/city-item-list.tsx

```diff
@@ -0,0 +1,24 @@
+import { useAppSelector } from '../../hooks/useAppDispatch';
+import { City } from '../../types/types';
+import { CityItem } from '../city-item/city-item';
+
+type CityItemListProps = {
+  cities: City[];
+};
+
+export const CityItemList = ({ cities }: CityItemListProps): JSX.Element => {
```

Comment:
Можно просто 
```suggestion
export const Cities = ({ cities }: CityItemListProps): JSX.Element => {
```

зачем усложнять абстрактными понятиями типа item, которые не несут никакой информации?